test/HouseRegistryExtV2.ts

```diff
@@ -5,88 +5,106 @@ import { ethers, upgrades } from 'hardhat';
+  // });
 
-  it('should return Transactions succesful', async function () {
-    await daiToken.connect(acc3).approve(houseRegistryExtV2.address, 200);
-    const func = await houseRegistryExtV2.connect(acc3).buyHouseWithDai(houseId);
-    const result = await func.wait();
-    await expect(result.events[2].args[0]).to.equal('Transactions succesful');
-    await expect(result.events[2].args[1]).to.equal(acc3.address);
-    await expect(await daiToken.connect(acc3).balanceOf(acc3.address)).to.equal(99999897);
-    await expect(await daiToken.connect(acc3).balanceOf(acc2.address)).to.equal(103);
-  });
+  // it('should return Transactions succesful', async function () {
+  //   const HouseRegistryExtV2 = await ethers.getContractFactory('HouseRegistryExtV2');
+  //   await upgrades.upgradeBeacon(beacon, HouseRegistryExtV2);
+  //   houseRegistryExtV2 = HouseRegistryExtV2.attach(houseRegistryExt.address);
+  //   await daiToken.connect(acc3).approve(houseRegistryExtV2.address, 200);
+  //   const func = await houseRegistryExtV2.connect(acc3).buyHouseWithDai(houseId);
+  //   const result = await func.wait();
+  //   await expect(result.events[2].args[0]).to.equal('Transactions succesful');
+  //   await expect(result.events[2].args[1]).to.equal(acc3.address);
+  //   await expect(await daiToken.connect(acc3).balanceOf(acc3.address)).to.equal(99999897);
+  //   await expect(await daiToken.connect(acc3).balanceOf(acc2.address)).to.equal(103);
+  // });
 
-  it('should return succesful', async function () {
-    const func = await houseRegistryExtV2.buyHouseWithETH(houseId, { value: 200 });
-    const result = await func.wait();
-    await expect(result.events[0].args[0]).to.equal('Transactions succesful');
-  });
+  // it('should return succesful', async function () {
+  //   const HouseRegistryExtV2 = await ethers.getContractFactory('HouseRegistryExtV2');
+  //   await upgrades.upgradeBeacon(beacon, HouseRegistryExtV2);
+  //   console.log(houseRegistryExt.address)
+  //   houseRegistryExtV2 = HouseRegistryExtV2.attach(houseRegistryExt.address);
+
+  //   const func = await houseRegistryExtV2.buyHouseWithETH(houseId, { value: 200 });
+  //   const result = await func.wait();
+  //   await expect(result.events[0].args[0]).to.equal('Transactions succesful');
+  // });
 
   it('should update contract and return houseId from first contract', async function () {
-    await houseRegistryExtV2.connect(acc3).listHouseSimple(100, 100, 100, 'asd');
-    await houseRegistryExtV2.connect(acc1).listHouseSimple(101, 101, 101, 'asd');
+    const HouseRegistryExtV2 = await ethers.getContractFactory('HouseRegistryExtV2');
+    await upgrades.upgradeBeacon(beacon, HouseRegistryExtV2);
+    houseRegistryExtV2 = HouseRegistryExtV2.attach(houseRegistryExt.address);
     const func = await houseRegistryExtV2.getExpensiveHouseIds();
-    await expect(func).to.equal(houseId);
+    await expect(await func).to.equal(houseId);
   });
```

Comment:
Here you're missing one check "obtain the house by the house ID from HouseRegistryExtV2 and make sure it's the same house as was already listed. Ex. check that the address is the same"